bin/shjs

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env node
-require('../global');
 
-if (process.argv.length < 3) {
-  console.log('ShellJS: missing argument (script name)');
+function exit(msg) {
+  console.log(msg);
   console.log();
   process.exit(1);
 }
 
+if (require.main !== module) {
```

Comment:
nit: can you move this up? Also, I assume you moved `require('../global')` down so that it doesn't pollute the test--this is OK, but you need to add a comment explaining why.